### ISSUE TITLE
Overriding second element of vector with arithmetic causes stack size exceeded

### ISSUE THREAD

**Issue by k-qy** [penrose/penrose#595]
Style repro is something like: `override factor.position[1] = x.position[1] + 1.0`

Happens on either side of the operator.

Reported by P3 in domain expert study on 6/11/21

**Comment by joshpoll** [comment]
There must be more going on to produce this error.

I tried adding the following to `linear-algebra-paper-simple.sty` and running the linear algebra example diagram:
```
forall VectorSpace U {
  U.origin2 = (0., 20.)
  override U.origin[1] = U.origin2[1] + 1.0
}
```

This runs without error.

**Comment by samestep** [comment]
I think this should be closed as of #984, because we no longer support `override` on access paths.
